ComponentKit/Core/Scope/CKDetectComponentScopeCollisions.mm

```diff
@@ -31,9 +31,53 @@
 }
 #pragma clang diagnostic pop
 
-void CKDetectComponentScopeCollisions(const CKComponentLayout &layout)
+static CKComponent *completeComponentScopeCollisionPair(CKComponent *collidingComponent,
+                                            id<NSObject> collidingScope,
+                                            NSMapTable<CKComponent *, CKComponent*> *componentsToParentComponents)
+{
+  for (CKComponent* componentKey in componentsToParentComponents) {
```

Comment:
Nit: Keep consistent with `*` placement. There should always be a space between the class and the asterisk.

```
for (CKComponent *componentKey...) {
```